model/registry/http.yaml

```diff
@@ -95,6 +95,12 @@ groups:
           was the cause of the resending (e.g. redirection, authorization failure, 503 Server Unavailable, network issues,
           or any other).
         examples: 3
+      - id: request.size
+        type: int
+        brief: >
+          The total size of the request in bytes. This should be the total number of bytes sent over the wire, and includes all framing and control
+          data, headers, and request body if any.
```

Comment:
what are framing and control data? thanks

Reply:
framing is mostly relevant to HTTP/2 and HTTP/3: those use streams that have a few different frame types, each of which has some header info.

control data is the request line and response line in HTTP/1.1 and is represented as pseudo-headers like :method and :authority in HTTP/2 and HTTP/3.